app/components/UI/QRCode.js

```diff
@@ -93,15 +95,30 @@ class ZapQRCode extends React.PureComponent {
     }
     if (size === 'large') {
       size = 180
+      border = 2
+    }
+    if (size === 'xlarge') {
```

Comment:
Consider extracting this into a constant. `xlarge` is also used down below

Reply:
Have extracted all preset size strings to constants.